recipe/meta.yaml

```diff
@@ -30,6 +30,7 @@ requirements:
     - libpng    # [build_platform != target_platform]
     - zlib      # [build_platform != target_platform]
   host:
+    - gobject-introspection
```

Comment:
This should not be necessary, and should be avoided at all cost to prevent conflicts between meson and gobject-introspection.

Reply:
It is needed in cross compiling, otherwise pkg-config will not find it. Not sure why

Reply:
In that case please add `# [build_platform != target_platform]` to it. Thanks